### DISCUSSION TITLE
Coding

### DISCUSSION THREAD

**Discussion by 56628426** [community/community#42024] [Mobile]
### Select Topic Area

General

### Body

Am a new here but am  not understand after make it positories so how to work on  it

**Comment by elli0t43** [comment]
Once you done creating your respository on github, you can either clone it locally then work on it or you can use github codespaces to work in that specifc respository. to get a better idea refer [here](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository).

for a headstart you can either use `git clone yourgithubrepo.git` or you can simply use the SSH method `git@github.com:yourusername/yourrespository.git` to clone your repository locally.

Now to add files you can refer [here](https://docs.github.com/en/repositories/working-with-files/managing-files/creating-new-files).

**Comment by jfullstackdev** [comment]
GitHub repos are just like regular folders in Google Drive, except, it has the version tracking that is so efficient

why do we need version control ? even if you are working alone in a project, you will be needing to know the changes, even the deletions of a certain part of code because that's very important

GitHub was based on that version control Git. GitHub is just the central space for everyone so we can collaborate here in just one place.

**Comment by longphu168** [comment]
> ### Select Topic Area
> 
> 
> 
> General
> 
> 
> 
> ### Body
> 
> 
> 
> Am a new here but am  not understand after make it positories so how to work on  it 
> 
>
